index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav id="navbar">
+        <ul>
+          <li>
+            <a href="">About</a>
+          </li>
+          <li>
+            <a href="#projects">Work</a>
+          </li>
+          <li>
+            <a href="#contact">contact</a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
- [ ] wrap the `nav` element inside the `header` element.

Reply:
It would sound much better if you expressed the same thought but in a form of a suggestion because this one sounds like an order. You should always explain why such a change is required. Example:
> You should wrap this `<nav>` element inside a `<header>` tag because it is a part of the top navigation bar which is supposed to be inside of a `<header>` tag.